specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/virtualRouter.json

```diff
@@ -0,0 +1,699 @@
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "virtualRouterName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Virtual Router."
+          },
+          {
+            "name": "peeringName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Virtual Router Peering."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/VirtualRouterPeering"
+            },
+            "description": "Parameters supplied to the create or update Virtual Router Peering operation."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Request received successfully. The operation returns the resulting Virtual Router Peering resource.",
+            "schema": {
+              "$ref": "#/definitions/VirtualRouterPeering"
+            }
+          },
+          "200": {
+            "description": "Request successful. The operation returns the resulting Virtual Router Peering resource.",
+            "schema": {
+              "$ref": "#/definitions/VirtualRouterPeering"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create Virtual Router Peering": {
+            "$ref": "./examples/VirtualRouterPeeringPut.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualRouters/{virtualRouterName}/peerings": {
+      "get": {
+        "tags": [
+          "VirtualRouterPeerings"
+        ],
+        "operationId": "VirtualRouterPeerings_List",
+        "description": "Lists all Virtual Router Peerings in a Virtual Router resource.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "virtualRouterName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Virtual Router."
```

Comment:
consider adding parameter definition for these too.. (similar to ApiVersionParameter)